contrib/gitian-keys/keys.txt

```diff
@@ -25,3 +25,4 @@ ED9BDF7AD6A55E232E84524257FF9BDBCC301009 Sjors Provoost
 77E72E69DA7EE0A148C06B21B34821D4944DE5F7 Nils Schneider
 79D00BAC68B56D422F945A8F8E3A8F3247DBCBBF Willy Ko
 AEC1884398647C47413C1C3FB1179EB7347DC10D Warren Togami
+D35176BE9264832E4ACA8986BF0792FBE95DC863 fivepiece
```

Comment:
Please sort by name, otherwise this will lead to issues in the future

Reply:
Done.  (it was not apparent that these are sorted at first)

Reply:
Should have sorted by int(key id) *ducks*

Reply:
That would make fingerprint fun just to be on top :)